react/features/app/actions.js

```diff
@@ -73,16 +75,13 @@ export function appNavigate(uri: ?string) {
         const { contextRoot, host, room } = location;
         const locationURL = new URL(location.toString());
 
-        // Disconnect from any current conference.
-        // FIXME: unify with web.
-        if (navigator.product === 'ReactNative') {
```

Comment:
I suspect this would break the case in which you follow a deep-link to a conference while already in a conference. I don't see how you'd disconnect from the first one.